lib/generators/rolemodel/linters/rubocop/rubocop_generator.rb

```diff
@@ -1,7 +1,10 @@
+# frozen_string_literal: true
+
 require_relative '../../../bundler_helpers'
 
 module Rolemodel
   module Linters
+    # Install Standard rubocop and custom Cop
```

Comment:
Unneeded capitalization
```suggestion
    # Install the standard rubocop and a custom cop
```